osu.Game/Beatmaps/BeatmapDifficultyManager.cs

```diff
@@ -307,5 +307,19 @@ public StarDifficulty(double stars, int maxCombo)
 
             // Todo: Add more members (BeatmapInfo.DifficultyRating? Attributes? Etc...)
         }
+
+        public DifficultyRating DifficultyRating
+        {
+            get
+            {
+                if (Stars < 2.0) return DifficultyRating.Easy;
```

Comment:
where are these values coming from? osu-web? may be good to leave an inline comment linking it up.